Cargo.toml

```
@@ -17,6 +17,9 @@ license = "GPL-3.0-only"
 edition = "2021"
 
 [workspace.dependencies]
+common = { path = "./common" }
```

```suggestion
supermusr-common = { path = "./common" }
```

Prefixing these would be a good idea to make it perfectly clear what they are.